iceoryx_posh/source/roudi/port_manager.cpp

```diff
@@ -274,7 +276,7 @@ void PortManager::handleInterfaces() noexcept
         {
             for (auto& instance : x.second.instanceSet)
             {
-                caproMessage.m_serviceDescription = capro::ServiceDescription(x.first, instance, capro::AnyEventString);
+                caproMessage.m_serviceDescription = capro::ServiceDescription(x.first, instance, roudi::Wildcard);
```

Comment:
Why not call it `AnyIdString`? What is the scope of use of the `wildcard`? Only for strings?